raster-foundry/raster-foundry
# Remove unused metrics collection resources

## Overview

Remove metrics collection resources that are no longer being used, cleaning them out of the dev environment:

* Strip out extraneous Docker Compose file for metrics
* Remove StatsD configs
* Adjust Vagrantfile and STRTA to remove references to metrics

### Checklist


## Notes

* This PR doesn't touch the Graphite server introduced in https://github.com/raster-foundry/raster-foundry/pull/4398. Judging by the timing of that PR coming in and #4400 getting filed, I assume that this is the only monitoring resource that we want to keep right now.

## Testing Instructions

* Run `vagrant up --provision` to update the VM
* Shell into the VM with `vagrant ssh`
* Run `./scripts/update` to update containers
* Run `./scripts/server` to start the development server
* Poke around and confirm that basic site functionality loads as expected (i.e. isn't blocked by Kamon tracing that isn't getting dumped anywhere)

Connects #4400
